PostProcessing/Runtime/PostProcessRenderContext.cs

```diff
@@ -90,14 +104,93 @@ public Camera camera
+            modifiedDesc.volumeDepth = m_sourceDescriptor.volumeDepth;
+            modifiedDesc.vrUsage = m_sourceDescriptor.vrUsage;
+            modifiedDesc.msaaSamples = m_sourceDescriptor.msaaSamples;
+            modifiedDesc.memoryless = m_sourceDescriptor.memoryless;
+
+            modifiedDesc.useMipMap = m_sourceDescriptor.useMipMap;
+            modifiedDesc.autoGenerateMips = m_sourceDescriptor.autoGenerateMips;
+            modifiedDesc.enableRandomWrite = m_sourceDescriptor.enableRandomWrite;
+            modifiedDesc.shadowSamplingMode = m_sourceDescriptor.shadowSamplingMode;
+
+            if (colorFormat != RenderTextureFormat.Default)
+                modifiedDesc.colorFormat = colorFormat;
+            if (readWrite != RenderTextureReadWrite.Default)
+                modifiedDesc.sRGB = (readWrite != RenderTextureReadWrite.Linear);
+
+            return modifiedDesc;
+        }
+#endif
+
+        public void GetScreenSpaceTemporaryRT(CommandBuffer cmd, int nameID, 
+                                            int depthBufferBits = 0, RenderTextureFormat colorFormat = RenderTextureFormat.Default, RenderTextureReadWrite readWrite = RenderTextureReadWrite.Default,
+                                            FilterMode filter = FilterMode.Bilinear, int widthOverride = 0, int heightOverride = 0)
+        {
+#if UNITY_2017_2_OR_NEWER
+            var desc = GetDescriptor(depthBufferBits, colorFormat, readWrite);
+            if (widthOverride > 0)
+                desc.width = widthOverride;
+            if (heightOverride > 0)
+                desc.height = heightOverride;
+
+            cmd.GetTemporaryRT(nameID, desc, filter);
+#else
+            int actualWidth = width;
+            int actualHeight = height;
+            if (widthOverride > 0)
+                actualWidth = widthOverride;
+            if (heightOverride > 0)
+                actualHeight = heightOverride;
+
+            cmd.GetTemporaryRT(nameID, actualWidth, actualHeight, depthBufferBits, filter, colorFormat, readWrite);
+            // TODO: How to handle MSAA for XR in older versions?  Query cam?
+            // TODO: Pass in vrUsage into the args 
+#endif
+        }
+
+        public RenderTexture GetScreenSpaceTemporaryRT(int depthBufferBits = 0, RenderTextureFormat colorFormat = RenderTextureFormat.Default, 
+                                                        RenderTextureReadWrite readWrite = RenderTextureReadWrite.Default, int widthOverride = 0, int heightOverride = 0)
+        {
+#if UNITY_2017_2_OR_NEWER
+            var desc = GetDescriptor(depthBufferBits, colorFormat, readWrite);
+            if (widthOverride > 0)
+                desc.width = widthOverride;
+            if (heightOverride > 0)
+                desc.height = heightOverride;
+
+            return RenderTexture.GetTemporary(desc);
+#else
+            int actualWidth = width;
+            int actualHeight = height 
```

Comment:
Missing semicolon :D 
`int actualHeight = height;`

Reply:
Someone else pointed this to me as well!  How the heck did this ever work?  Thanks, will fix!